front-end/js/script.js

```diff
@@ -1,64 +1,90 @@
+var places = [];
+var markers = [];
+var map;
+
```

Comment:
Regarding the file name 'script.js' can we be more specific?

Reply:
ok!